### DISCUSSION TITLE
Tips to use it on constrained containers

### DISCUSSION THREAD

**Discussion by hjJunior** [tldraw/tldraw-v1#564] [General]
Hello everyone, first thanks for this amazing project 👏 

I'm playing with the project, and got some questions, specifically about how to run in constrained containers (limited size)

## Example
![image](https://user-images.githubusercontent.com/10780132/154330881-6dfc994b-5cc1-431c-b21b-e689c7741070.png)

I have a layout, where blue shapes are fixed elements, and the part of TLDraw have some content, which I would like to enable the users to draw over it

I noticed, the TLDraw uses position absolute to fit the entire window, and my objective is make this to work on small container where I have limited width /or height (not the entire screen), so I have done some layout adjustments.

But I'm facing some issues when drawing / or selecting, which the pointer is not aligned from where I actually started the mouse cursor (seems be caused due the differences of contrained container)

### Represetantion
![image](https://user-images.githubusercontent.com/10780132/154332377-3cb929bd-4bf0-44a4-9b54-24683a5c144b.png)

- **purple dot** Where I started the click event to draw
- **white dot** Where actually is started the drawing element

I would love to have some tips how to fix this, any help will be appreciated

**Comment by steveruizok** [comment]
Hey, you should be able to put the container anywhere on the page and have it work. Could you send me a link if you're able to reproduce the bug you're describing?
